Utilities/bootstrap

```diff
@@ -755,10 +755,10 @@ def main():
     parser.add_argument("--build", dest="build_path",
                         help="create build products at PATH [%(default)s]",
                         default=".build", metavar="PATH")
-    parser.add_argument("--prefix", dest="install_prefix",
-                        help="use PATH as the prefix for installing "
+    parser.add_argument("--prefix", dest="install_prefixes", nargs='*',
+                        help="use PATHS as the prefixes for installing "
                              "[%(default)s]",
-                        default="/usr/local", metavar="PATH")
+                        default=["/usr/local"], metavar="PATHS")
```

Comment:
This change means that we will always install in "/usr/local", because we just append to the default. I think you need this array to start as empty, and only apply the default if none was specified.